### ISSUE TITLE
Change in hostname ?

### ISSUE THREAD

**Issue by grixtil** [vagrantpress/vagrantpress#104]
Hi there,

I like this vagrant Wordpress project, but would like to change the hostname of the project as I have multiple projects running on my machine at the same time (I'm developing multiple wordpress sites for a project), and it gets annoying to have to suspend and up again different vagrant instances constantly.

I've tried changing the host file, but I get an "Application not found" from Pow.  Could you please guide me on how / where I am able to change this so that when I init a new instance, it would be configured with it's own hostname rather than the generic one?

Thanks

**Comment by fredrikekelund** [comment]
Something that would be nice would be for instance a YAML file that works as a single way of configuring the hostname and maybe a set of directories that are synced in the Vagrantfile.

**Comment by superpowered** [comment]
I do this on all my projects.
The process is pretty simple if you have an IDE that can do a "Replace All" in the entire directory. It's pretty much just changing two variables.

1. Do a replace all for "vagrantpress.dev" and replace it with "vagrantpress._XXXX_"
2. Do a replace all for "192.168.50.4" and replace it with "192.168.50._XX_"
3. Run vagrant up.

**Comment by fredrikekelund** [comment]
Yeah, I've done the same, and it works just fine, but it doesn't *feel* like official support. But on the other hand, the good thing about this project is how lean and focused it is. Wouldn't want to spoil that either

**Comment by dskvr** [comment]
It seems practical that the vagrantfile, or file responsible for setting the host (haven't checked), could conditionally check for a parameter set by environment variable when running `vagrant up` ([link](http://stackoverflow.com/questions/14124234/how-to-pass-parameter-on-vagrant-up-and-have-it-in-the-scope-of-vagrantfile)).
